object_model/literal.py

```diff
@@ -33,7 +42,7 @@ def __init__(self, v, name, xp_state):
         self.xp_state.literalNameToObj[self.name] = self
         self.xp_state.ghostFiles |= {self.loc, }
 
-    def forEach(self):
+    def __forEach__(self):
```

Comment:
This would be a good place to set the default. So, in the constructor we assume that self.v is the default. And if __oneByOne__ is called, then we set the default to self.v[0]

Reply:
Right this makes sense for the forEach literals, but would we set the defaults of the other type of literals in the constructor? Lets talk about this in the meeting